src/test/java/org/fcrepo/camel/integration/FcrepoTransactionIT.java

```diff
@@ -48,39 +48,40 @@
  * @author Aaron Coburn
  * @since November 7, 2014
  */
-public class FcrepoTransactionIT extends CamelTestSupport {
+@Ignore
```

Comment:
Might be good to add a TODO here to remind us we need to update this test to work with fcrepo6 transactions.